dbt_copilot_helper/templates/pipeline/manifest.yml

```diff
@@ -0,0 +1,39 @@
+# {% version_info %}
+# This YAML file defines your pipeline: the source repository it tracks and the order of the environments to deploy to.
+# For more info: https://aws.github.io/copilot-cli/docs/manifest/pipeline/
+
+# The name of the pipeline.
+name: {{ app_name }}-environments
+
+# The version of the schema used in this template.
+version: 1
+
+# This section defines your source, changes to which trigger your pipeline.
+source:
+  # The name of the provider that is used to store the source artifacts.
+  # (i.e. GitHub, Bitbucket, CodeCommit)
+  provider: GitHub
+  # Additional properties that further specify the location of the artifacts.
+  properties:
+    # Todo: Allow for overriding this, but without risking deploying a branch to higher environments
+    branch: main
+    repository: {{ git_repo }}
+    # Optional: specify the name of an existing CodeStar Connections connection.
+    connection_name: {{ app_name }}
+
+# This section defines the order of the environments your pipeline will deploy to.
+stages:
+  {% for environment in pipeline_environments -%}
```

Comment:
For nicer yaml formatting, can we use an indent of 0 on for loops?

Reply:
Done